src/backend/joanie/tests/signature/test_backend_signature_dummy.py

```diff
@@ -110,6 +110,84 @@ def test_backend_dummy_signature_get_signature_invitation_link(self):
         # Check that an email has been sent
         self._check_signature_completed_email_sent("student_do@example.fr")
 
+    def test_backend_dummy_signature_get_signature_invitation_link_for_organization(
+        self,
+    ):
+        """
+        Dummy backend instance get_signature_invitation_link method should return an
+        invitation link to sign the contract.
+
+        If the contract has been signed by the student, calling this method should send
+        an email to the organization signatory and call the handle notification method
+        to mimic the fact that the organization has signed the contract.
+        """
+        backend = DummySignatureBackend()
+        expected_substring = "https://dummysignaturebackend.fr/?requestToken="
+        reference, file_hash = backend.submit_for_signature(
+            "title definition 1", b"file_bytes", {}
+        )
+        contract = factories.ContractFactory(
+            signature_backend_reference=reference,
+            definition_checksum=file_hash,
+            submitted_for_signature_on=django_timezone.now(),
+            student_signed_on=django_timezone.now(),
+            context="a small context content",
+        )
+
+        response = backend.get_signature_invitation_link(
+            recipient_email="student_do@example.fr",
+            reference_ids=[reference],
+        )
+
+        self.assertIn(expected_substring, response)
+
+        contract.refresh_from_db()
+        self.assertIsNotNone(contract.student_signed_on)
+        self.assertIsNotNone(contract.organization_signed_on)
+        self.assertIsNone(contract.submitted_for_signature_on)
```

Comment:
Should this be instead : 
```suggestion
        self.assertIsNotNone(contract.submitted_for_signature_on)
```
According to the model comment, it should be set once the document has been sent for a signature process. Are we still using this logic ? maybe I've missed something. Here is a snippet of the code from the `Contract` Model :

```python
    # Set when contract is sent to signature provider
    submitted_for_signature_on = models.DateTimeField(
        _("Date and time we send the contract to signature provider"),
        null=True,
        blank=True,
        editable=False,
    )
```

Reply:
Yup we are still using this logic but when the contract has been fully signed, this property is sets to None.

Reply:
Okay for me then, I've missed this part where we set it None after the completion of signature. 
:rocket: let's go